package.json

```diff
@@ -31,12 +31,16 @@
     "fixtures-fs": "^2.0.0",
     "istanbul": "~0.3.2",
     "jshint": "2.5.6",
+    "mkdirp": "^0.5.1",
     "pre-commit": "0.0.9",
     "tap-spec": "~1.0.0",
     "tape": "^3.0.2"
   },
   "scripts": {
+    "pretest": "rm -fr test/proj /tmp/proj",
```

Comment:
please dont use pre scripts.

The correct way is to add a `rimraf('./proj'); rimraf('/tmp/proj')` to the source code.